dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -1,5 +1,7 @@
 FROM docker-config-engine
 
+RUN echo 'deb [arch=amd64] http://packages.microsoft.com/repos/sonic-dev/ jessie main' >> /etc/apt/sources.list
```

Comment:
>sonic-dev/ [](start = 63, length = 10)

Why you need sonic-dev repo?

Reply:
sonic-dev repo is necessary to download python-click and python-tabulate packages which are dependencies of sonic-utilities. This can all be removed in the future once this uses the PSU plugin directly (see my comment above).

Reply:
Is it possible to install them by pip?

Reply:
Unfortunately, dependency issues arise when we have Debian packages that rely on Python packages and vice-versa (for example: https://github.com/Azure/sonic-utilities/pull/124).